### PR TITLE
EditingToolkit > GlobalStyles: Add new Google fonts

### PR DESCRIPTION
**Pull request by allilevine**
#### Changes proposed in this Pull Request

We've decided to offer additional Google fonts on WordPress.com. This PR adds those fonts to the Editing Toolkit > Global Styles > Font Selection options:

Screen Shot 2020-08-10 at 3 20 51 PM

#### Testing instructions

To test this change on your sandbox you can apply the patch D47730-code and sandbox the public API, then test on your sandboxed site.
- Edit a page and click the "A" button in the upper right.
- [ ] The following fonts are available under "Font Selection" in both the Heading Font and Base Font select elements:
Theme Default
System Font
Arvo
Cabin
Chivo
Courier Prime
Domine
EB Garamond
Fira Sans
Josefin
Libre Baskerville
Libre Bodoni
Libre Franklin
Lora
Merriweather
Montserrat
Nunito
Open Sans
Overpass
Playfair Display
Poppins
Raleway
Roboto
Roboto Slab
Rubik
Source Sans Pro
Source Serif Pro
Space Mono
Work Sans

Fixes https://github.com/Automattic/dotcom-manage/issues/111

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php [hunk 1]
```diff
@@ -93,14 +93,20 @@ class Global_Styles {
 		'Arvo',
 		'Cabin',
 		'Chivo',
+		'Courier Prime',
 		'Domine',
+		'EB Garamond',
 		'Fira Sans',
+		'Josefin',
 		'Libre Baskerville',
+		'Libre Bodoni',
```
Comment (allilevine) [inline apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php:102]: This isn't a font Google offers so I'll need to remove or replace it.

**Comment by glendaviesnz** [comment]
I tested this and was able to select the fonts in the editor, but when saved the fonts were not applied in the front end ... should this fix also apply the fonts to the front end view?

Screen Shot 2020-08-11 at 2 41 23 PM

**Comment by glendaviesnz** [comment]
This tested well for me ... happy to give it a green tick tomorrow once you decide what to do with that font ... I would be inclined to remove it for now and merge others, unless you can track down an alternative one quickly.

**Comment by allilevine** [comment]
> This tested well for me ... happy to give it a green tick tomorrow once you decide what to do with that font ... I would be inclined to remove it for now and merge others, unless you can track down an alternative one quickly.

Thanks! I removed Libre Bodoni. I also corrected the name of "Josefin" to "Josefin Sans".

**Comment by glendaviesnz** [comment]
given that this doesn't get merged into the calypso app you can probably ignore the e2e test failures

**Comment by simison** [comment]
Completely unrelated to this PR but shouldn't these be wrapped in translation functions?

https://github.com/Automattic/wp-calypso/blob/493bac8ebce3ec23344643e013ee4f3482c53c91/apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php#L84-L91
https://github.com/Automattic/wp-calypso/blob/493bac8ebce3ec23344643e013ee4f3482c53c91/apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php#L191
https://github.com/Automattic/wp-calypso/blob/493bac8ebce3ec23344643e013ee4f3482c53c91/apps/full-site-editing/full-site-editing-plugin/global-styles/class-global-styles.php#L147

**Comment by allilevine** [comment]
> Completely unrelated to this PR but shouldn't these be wrapped in translation functions?

@simison Thanks for pointing that out! They should be. Can we open a new issue to add translation functions? I don't want to hold up the font updates.

**Comment by pablinos** [comment]
This is working for me too. I think we should get some help with deployment. I'm not sure we've got access to the .org repo for example.

> Can we open a new issue to add translation functions?

Yes, let's do that as a separate PR. Let's open an issue so we don't forget.

**Pull request merged**